package.json

```diff
@@ -68,6 +68,7 @@
     "proxy-polyfill": "0.1.6",
     "re-reselect": "^4.0.0",
     "react-beautiful-dnd": "^13.1.0",
+    "react-color": "^2.19.3",
     "react-datetime-picker": "^3.4.3",
     "react-plastic": "^1.1.1",
     "react-top-loading-bar": "^2.0.1",
```

Comment:
- As a creator, when I Heart someone, it uses the correct icon, but when I unheart, it becomes a solid red heart instead of reverting to solid white.  Scrolling a bit fixes it.
- Should it be solid white in the first place?  Given that the rest are hollow, a hollow heart would be better choice for "not yet clicked", while the solid one seems to imply "it's already clicked".
![image](https://user-images.githubusercontent.com/64950861/150709172-e53accc8-3616-4bb4-a86c-2b73eec245f4.png)

Reply:
Fixed your 2nd point, but I'm unable to reproduce the 1st one. I wasn't able to test it on the notification page tho.